validate_tag.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+#
+# Set of Regular expressions / checks to ensure compliance of git version tags with SODALITE standards
+#
+# <valid prerelease tag> ::= <version core> "-" <pre-release>
+#                          | <version core> "-" <pre-release> "+" <build>
+#                          | <major release>
+#
+# <valid release tag> ::= <version core>
+#                       | <version core> "+" <build>
+#                       | <major release>
+#
+# <major release> ::= "M18Release" | "M24Release" | "M36Release"
+#
+# <version core> ::= <major> "." <minor> "." <patch>
+
+# Usage:
+#   $0 [SemVar|SemVarStage|SemVarProd|MajRel|production|staging] <tag>
+
+FUNC=$1
+VALUE=$2
+
+# Functions
+
+SemVar() {
+  if [[ "$VALUE" =~ ^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-((0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*))*))?(\+([0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*))?$ ]]; then
+    echo true | tr -d '\n'
```

Comment:
```suggestion
    echo -n true
```

Reply:
I'm inclined not to change this -- it is identical to/copied from code in other repositories for the same purpose.